docs/developer/release/qa.md

```diff
@@ -195,3 +195,25 @@ should point the user to the Qubes notifications in the top-right corner:
 3. The `dz-dvm` disposable Qube cannot start due to insufficient resources. We
    can trigger this scenario by temporarily increasing the minimum required RAM
    of the `dz-dvm` template to more than the available amount.
+
+## Large Document Testing
+
+Parallel to the QA process, the release candidate should be put through the large document tests in a dedicated machine to run overnight.
+
+Follow the instructions in `docs/developer/TESTING.md` to run the tests.
+
+These tests will identify any regressions or progression in terms of document coverage.
+
+## Scripted QA
+
```

Comment:
I think this section would be better upper in the doc, so that people read it first, especially as it allows them to get some time back.

Reply:
We are actually referring to `dev_scripts/qa.py` at the top of the docs. It's just that in this section, we add some rationale about the tool, which does not need to go at the very top. Does that make sense?

Reply:
Since it's just a few lines, I think we could factorize it to the top without impeding the reading. Anyway, use you best judgement, I'm with you :-)